radian/app.py

```diff
@@ -75,35 +75,35 @@ def main(cleanup=None):
     if not r_home:
         raise RuntimeError("Cannot find R binary. Expose it via the `PATH` variable.")
 
-    if sys.platform.startswith("linux"):
```

Comment:
I think we should still restrict ourself to non-window.

For example 
```
 if not sys.platform.startswith("win"):
```

Reply:
Agreed yeah that makes sense

Reply:
Do you want to do the edit?

Reply:
@randy3k Sure let me add that, 1 sec